docs/API.md

```diff
@@ -170,7 +170,7 @@ Basic data type: object
 Properties:
 
 * `"ns"`: A *domain name*, required.
-* `"ip"`: An *IP address*, IPv4 or IPv6 , optional.
+* `"ip"`: An *IP address*, IPv4 or IPv6 , optional. (default: `undef`)
```

Comment:
undef is the Perl value when a variable is not set.
We can change to null which is more generic.

Reply:
The reason we should have `null` here is because it's a JSON-RPC API, so we should refer to the JSON types.

Reply:
@pamasse Another thing I forgot to mention. This may be nitpicking, but could you also update it so it says "An IP address or null"? I think that would be really neat.

Reply:
done